src/components/elements/card.js

```diff
@@ -7,7 +7,6 @@ import { LocalizedLink } from 'components/localization'
 import device from 'themes/device'
 // SVG
 import Arrow from 'images/svg/elements/card-arrow.svg'
-import Diagonal from 'images/svg/elements/pink-right-diagonal.svg'
```

Comment:
Can we remove the `pink-right-diagonal.svg` file as well if it's not used anywhere else?

Reply:
sure @mohammad-rework